<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolutionWindow</class>
 <widget class="QMainWindow" name="SolutionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="solutionTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Interaction Surface</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="GLWidget" name="glwidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mx - My</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsViewMxMy">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>611</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNValue">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interaction Curve for N [kN]:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDrawForN">
      <property name="geometry">
       <rect>
        <x>333</x>
        <y>11</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>My - N</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsViewNMy">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>611</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>6</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Interaction Curve for Mx [kNm]:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMxValue">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDrawForMx">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Mx - N</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsViewNMx">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>611</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interaction Curve for My [kNm]:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMyValue">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDrawForMy">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>M-N</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interaction Curve for alfa [degree]:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAlfaValue">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDrawForAlfa">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewNM">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>611</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonExport">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>560</y>
      <width>114</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Export To File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N values</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMx">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mx values</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMy">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>My values</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAll">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertices (Mx, My,N)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonImport">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>587</y>
      <width>114</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Import From File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
